website/docs/r/cloud_provider_snapshot_backup_policy.html.markdown

```diff
@@ -0,0 +1,118 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: cloud_provider_snapshot_backup_policy"
+sidebar_current: "docs-mongodbatlas-resource-cloud-provider-snapshot-backup-policy"
+description: |-
+    Provides an Cloud Provider Snapshot Backup Policy resource.
+---
+
+# mongodbatlas_cloud_provider_snapshot_backup_policy
+
+`mongodbatlas_cloud_provider_snapshot_backup_policy` provides a resource that enables you to view and modify the snapshot scheduling and retention settings for an Atlas cluster with Cloud Provider Snapshots enabled
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_cluster" "my_cluster" {
+  project_id   = "<PROJECT-ID>"
+  name         = "clusterTest"
+  disk_size_gb = 5
+
+  //Provider Settings "block"
+  provider_name               = "AWS"
+  provider_region_name        = "EU_CENTRAL_1"
+  provider_instance_size_name = "M10"
+  provider_backup_enabled     = true // enable cloud provider snapshots
```

Comment:
Update comment on this one to read  (again they might be able to adjust the actual policy without this being true - need to test - but it's doesn't have any meaning unless provider_backup_enabled is true)
// must be enabled in order to use cloud_provider_snapshot_backup_policy resource

Reply:
thank you :)